src/System.Management.Automation/security/nativeMethods.cs

```diff
@@ -1088,36 +1088,36 @@ internal static DWORD DestroyWintrustDataStruct(WINTRUST_DATA wtd)
         [StructLayout(LayoutKind.Sequential)]
         internal struct CRYPT_PROVIDER_CERT
         {
-            private DWORD _cbStruct;
+            private readonly DWORD _cbStruct;
```

Comment:
I am unsure about pInvoke code.  Let's not make any changes to any PInvoke in this PR.

Reply:
I am unsure about pInvoke code. Let's not make any changes to any PInvoke in this PR.